packages/neos-ui-editors/src/index.js

```diff
@@ -0,0 +1,96 @@
+import EditorEnvelope from './EditorEnvelope/index';
+
+import {
+    TextField,
+    TextArea,
+    Boolean,
+    DateTime,
+    Image,
+    SelectBox,
+    Link,
+    Reference,
+    References,
+    NodeType,
+    CodeMirror,
+    MasterPlugin,
+    AssetEditor,
+    PluginViews,
+    PluginView
+} from './Editors/index';
+
+import {
+    CodeMirrorWrap,
+    ImageCropper,
+    MediaDetailsScreen,
+    MediaSelectionScreen
+} from './SecondaryEditors/index';
+
+import AssetUpload from './Library/AssetUpload';
+import AssetOption from './Library/AssetOption';
+
+import LinkInput from './Library/LinkInput';
+import LinkOption from './Library/LinkOption';
+
+import NodeOption from './Library/NodeOption';
+
+export {
+    EditorEnvelope,
+
+    TextField,
+    TextArea,
+    Boolean,
+    DateTime,
+    Image,
+    SelectBox,
+    Link,
+    Reference,
+    References,
+    NodeType,
+    CodeMirror,
+    MasterPlugin,
+    AssetEditor,
+    PluginViews,
+    PluginView,
+
+    CodeMirrorWrap,
+    ImageCropper,
+    MediaDetailsScreen,
+    MediaSelectionScreen,
+
+    AssetUpload,
+    AssetOption,
+    LinkInput,
+    LinkOption,
+    NodeOption
+};
+
+export default {
```

Comment:
Why does need to be exported twice? Is there no way to avoid it somehow? Doesn't look DRY...

Reply:
I think I asked that question about a year ago or so and didn't get a good answer, since then I am just doing it that way (as in other parts of the projects). Back then I didn't find a way to make it work with just one.

Reply:
Duh ok if it's this way everywhere could also be fixed later...